data/json/monsters/mutant_mammal.json

```diff
@@ -151,35 +124,51 @@
   {
     "id": "mon_cat_mutant_kitten_prism",
     "type": "MONSTER",
-    "copy-from": "mon_cat_mutant_prism",
     "name": { "str": "iridescent kitten" },
     "description": "In the light, this adorably tiny bundle of unkempt fluff is an ever-changing flurry of brilliant colors.",
+    "copy-from": "mon_cat_kitten",
     "default_faction": "cat",
-    "bodytype": "dog",
-    "categories": [ "WILDLIFE" ],
-    "species": [ "MAMMAL" ],
-    "volume": "1000 ml",
-    "weight": "1 kg",
-    "hp": 6,
-    "material": [ "flesh" ],
-    "symbol": "c",
-    "color": "light_gray",
     "luminance": 4,
+    "//": "Higher dodge due to the constantly changing fur pattern",
+    "dodge": 15,
```

Comment:
I thought 10 was the cap, but I actually don't see that documented. Maybe there is no cap?

Reply:
I do not know. `mon_cat_kitten` has a dodge of 14, so I just +1'ed that.

Reply:
Works for now, I'll try and figure this out later and get it documented